### PR TITLE
Create Innovation-Specialist-GS-14.md

### PR DESCRIPTION
**Pull request by pixelwake**
To #127

### CODE REVIEW

**Comment by pixelwake** [comment]
Ready for review.

File: _positions/Innovation-Specialist-GS-14.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Innovation Specialist, GS-0301-14
+permalink: who-we-are-hiring/positions/Innovation-Specialist-GS-14/
```
Comment (fureigh) [inline _positions/Innovation-Specialist-GS-14.md]: See questions and comments on the corresponding section of #128.

File: _positions/Innovation-Specialist-GS-14.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Innovation Specialist, GS-0301-14
+permalink: who-we-are-hiring/positions/innovation-specialist-gs-14/
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-14.md:3]: Changed link to lowercase.

File: _positions/Innovation-Specialist-GS-14.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- Expert analytical skills to apply a wide range of qualitative and/or quantitative methods for assessing and improving program effectiveness
+
+Candidates are also evaluated for their competencies in key areas, including:
+- Analysis and creative problem solving: Expertise determining user needs or identifying market opportunities and designing and deploying solutions
+- Technology acumen: Knowledge of standard and emergent web technologies
+- Entrepreneurship and product management: Experience delivering products or services from concept to market
+- Approach and methodology: Expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and emotional intelligence: Experience in client facing roles, navigating bureaucracy, business development, or delivering services with a strong user facing or consumer oriented element
+- Tools and execution: Experience in design, development and deployment of solutions leveraging current and emergent technologies and best practices
```
Comment (pixelwake) [inline _positions/Innovation-Specialist-GS-14.md:41]: Deleted and re-added hyphens.

**Comment by pixelwake** [comment]
Final changes made, merging.

**Pull request merged**